test/packages/parallel/apache/data_stream/access/_dev/test/policy/test-default.expected

```diff
@@ -17,6 +17,7 @@ inputs:
             - apache-access
       type: logfile
       use_output: default
+namespaces: []
```

Comment:
It looks like that for policy tests it should also be needed to be able to mark some fields optional?
This `namespaces` field appeared when testing with `8.16.0-SNAPSHOT`.

Changes required present in these two commits:
- https://github.com/elastic/elastic-package/pull/2067/commits/ba7e1f5f05f4cae7ac91835a94800982dc62490b
- https://github.com/elastic/elastic-package/pull/2067/commits/83a8623bf19143627eec25bdf3b5c5f9bdaf6dfc

Reply:
Reverting these two commits in order to be able to merge this PR.